packaging/prerelease/build_app.sh

```diff
@@ -111,4 +113,6 @@ BUCKET_NAME="$bucket_name" PLATFORM="$platform" "$dir/s3_index.sh"
 
 "$client_dir/packaging/slack/send.sh" "Finished $platform $build_desc (keybase: $version, kbfs: $kbfs_version). See $s3host"
 
-BUCKET_NAME="$bucket_name" "$dir/report.sh"
+if [ "$istest" = "" ]; then
```

Comment:
I think the standard way to do this is `[ -z "$istest" ]`, but on the other hand your way doesn't require memorizing what `-z` means...I'm fine with either.